exercises/practice/complex-numbers/.meta/config.json

```diff
@@ -14,7 +14,8 @@
     "SleeplessByte",
     "Smarticles101",
     "sshine",
-    "Wasy18"
+    "Wasy18",
+    "jagdishdrp"
```

Comment:
I think this is changing in one of your other PRs. Could you remove this change from this PR?

Reply:
I’m not entirely sure how this ended up here, and I apologize for the oversight. I promise it won’t happen again.